adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -243,13 +243,9 @@ - (UIView *)renderBannerAdInView:(UIView *)bannerView
                                             placementID:placementID
                                                   error:&bannerError];
   if (success) {
-    self.isMrecPlaying = YES;
-
     return bannerView;
   } else {
     NSLog(@"Banner loading error: %@", bannerError.localizedDescription);
-    self.isMrecPlaying = NO;
```

Comment:
Remove self.isMrecPlaying = NO, because this could lead to the old MREC ad view can not be closed if the new MREC is created failed. 🙂

Reply:
I noticed that the issue (IOS-2491) can happen only when I hit close button on `TestBannerAdViewController` before we set `self.isMrecPlaying = YES;`. So, SDK keep trying to render ad. But QA test app already get back to main screen. So, moveing `self.isMrecPlaying = YES;` in `vungleDidShowAdForPlacementID` could make situation worse (since `vungleDidShowAdForPlacementID` will called later than this code.) We might need to stay as before?

Reply:
OK. Sounds good.